<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerfSettingsPage</class>
 <widget class="QWidget" name="PerfSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="perfPathLabel">
     <property name="text">
      <string>Perf Binary:</string>
     </property>
     <property name="buddy">
      <cstring>perfPathEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KUrlRequester" name="perfPathEdit">
     <property name="placeholderText">
      <string>perf</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
